Sources/Sentry/SentryScope.m

```diff
@@ -97,7 +97,7 @@ - (instancetype)initWithScope:(SentryScope *)scope
         [_fingerprintArray addObjectsFromArray:[scope fingerprints]];
         [_attachmentArray addObjectsFromArray:[scope attachments]];
 
-        self.propagationContext = [[SentryPropagationContext alloc] init];
+        self.propagationContext = scope.propagationContext;
```

Comment:
l: Wondering if we should keep the same pointer reference or also copy `propagationContext`.

Reply:
Good point, I made SentryPropagationContext readonly instead, so it doesn't matter.